cmd/bm-client/handlers/create_account.go

```diff
@@ -50,7 +66,7 @@ func CreateAccount(vault *vault.Vault, bmAddr, name, organisation, routing, toke
 		fmt.Printf("not found. This is a good thing.\n")
 
 		fmt.Printf("* Generating your secret key to send and read mail: ")
-		privKey, pubKey, err := encrypt.GenerateKeyPair(bmcrypto.KeyTypeRSA)
+		privKey, pubKey, err := bmcrypto.GenerateKeyPair(bmcrypto.KeyTypeRSA)
```

Comment:
- Should we ask for entropy input?
- Generate a deterministic RSA key based on BIP39 seed as a recovery phrase?

Reply:
we get entropy through crypto/rand.Reader, which in turn uses the correct random generators of the platform. It should be blocking for all platforms when not enough entropy can be found.